### ISSUE TITLE
Zoomable Lighttable Does Not Allow Navigation

### ISSUE THREAD

**Issue by kap5535** [darktable-org/darktable#18085]
### Describe the bug

The zoomable lightable allows me to zoom in and out but does not allow navigation by selecting the lighttable with the left mouse button. The lighttable does not move.

### Steps to reproduce

Got to lighttable and select zoomable lighttable mode.  Hold down the left mouse button and try to drag the lighttable around - it will not move.

### Expected behavior

darktable should allow dragging of the lighttable in zoomable lighttable mode

### Where did you obtain darktable from?

downloaded from www.darktable.org

### darktable version

5.0.0

### What OS are you using?

Linux

### What is the version of your OS?

Mint 22

### Describe your system?

System:
  Kernel: 6.8.0-51-generic arch: x86_64 bits: 64 compiler: gcc v: 13.3.0 clocksource: tsc
  Desktop: Cinnamon v: 6.2.9 tk: GTK v: 3.24.41 wm: Muffin v: 6.2.0 vt: 7 dm: LightDM v: 1.30.0
    Distro: Linux Mint 22 Wilma base: Ubuntu 24.04 noble
Machine:
  Type: Desktop System: ASUS product: N/A v: N/A serial: <superuser required>
  Mobo: ASUSTeK model: ProArt Z690-CREATOR WIFI v: Rev 1.xx serial: <superuser required>
    part-nu: SKU uuid: <superuser required> UEFI: American Megatrends v: 2602 date: 07/03/2023
CPU:
  Info: 12-core (8-mt/4-st) model: 12th Gen Intel Core i7-12700F bits: 64 type: MST AMCP
    smt: enabled arch: Alder Lake rev: 2 cache: L1: 1024 KiB L2: 12 MiB L3: 25 MiB
  Speed (MHz): avg: 800 high: 827 min/max: 800/4800:4900:3600 cores: 1: 800 2: 800 3: 800 4: 800
    5: 800 6: 800 7: 800 8: 800 9: 800 10: 800 11: 800 12: 800 13: 800 14: 790 15: 800 16: 800
    17: 800 18: 800 19: 800 20: 827 bogomips: 84480
  Flags: avx avx2 ht lm nx pae sse sse2 sse3 sse4_1 sse4_2 ssse3 vmx
Graphics:
  Device-1: NVIDIA GA106 [GeForce RTX 3060 Lite Hash Rate] vendor: ASUSTeK driver: nvidia
    v: 535.183.01 arch: Ampere ports: active: none off: DP-1,DP-2 empty: DP-3,HDMI-A-1,HDMI-A-2
    bus-ID: 0000:01:00.0 chip-ID: 10de:2504 class-ID: 0300
  Display: x11 server: X.Org v: 21.1.11 with: Xwayland v: 23.2.6 driver: X: loaded: nouveau
    unloaded: fbdev,modesetting,vesa failed: nvidia gpu: nvidia,nvidia-nvswitch display-ID: :0
    screens: 1
  

### Are you using OpenCL GPU in darktable?

Yes

### If yes, what is the GPU card and driver?

See above in system description

### Please provide additional context if applicable. You can attach files too, but might need to rename to .txt or .zip

Has been duplicated by others as per following thread
https://discuss.pixls.us/t/possible-dt-5-0-0-bugs/47254

**Comment by dmkUK** [comment]
I can confirm on an Ubuntu 22LTS install with darktable from the OBS using X11 and an AMD card with opencl on.

I tend to use the zoom-able light table. Previously you could hover over a particular thumbnail and use the mouse wheel to zoom in and out to it, eventually centring on a single thumbnail. You could also left click to grab the thumbnail(s) and drag them across the screen, very useful if your thumbnails are not all visible. The only way to move the light-table now appears to be with the cursor keys.

I thought this problem went further because whilst zoomed in in the darkroom I struggled to drag the main view around. The cursor only changed to a hand when left clicking around the edges of the view and changed to a pair of triangles to indicate the boundaries of these zones (which were of different widths). The centre of the screen was non interactive. Fortunately after toggling with the scroll bar setting, leaving and re-entering the darkroom view and closing and restarting darktable in an attempt to force a redraw of the screen, the problem cleared and has not resurfaced.

**Comment by gwenprime** [comment]
I can confirm this issue on both my arch linux machine as well as Windows 11. I wasn't sure if it was a bug or just a UX change but I definitely miss being able to drag around in the lighttable.

**Comment by vsz1lu** [comment]
I can confirm - 5.1.0~git23.8855ec9e-1+12385.1 Ubuntu 24.04

**Comment by helloworld1** [comment]
Looks like a regression tested on both Arch linux (Wayland) and Win 11

**Comment by vosbergw** [comment]
Same issue with:
Ubuntu 24.10
Asus NUC14RVS
Intel Ultra 5125H
Intel Arc Graphics (w/ CL support)
Wayland

**Comment by matt-hoskins** [comment]
Left-click-on-thumbnail-to-navigate isn't working for me either in the culling-layout-in-fixed-mode either - left clicking selects the thumbnail (the background to the thumbnail turns a lighter grey) but doesn't change what images are being displayed above (i.e. change in behaviour from 4.8).

**Comment by ptilopteri** [comment]
* matt-hoskins ***@***.***> [01-09-25 15:09]:
> Left-click-on-thumbnail-to-navigate isn't working for me either in the culling-layout-in-fixed-mode either - left clicking selects the thumbnail (the background to the thumbnail turns a lighter grey) but doesn't change what images are being displayed above (i.e. change in behaviour from 4.8).


<left> or <right><arrow> will change image in the thumbnail display.

-- 
(paka)Patrick Shanahan       Plainfield, Indiana, USA          @ptilopteri
facebook/ptilopteri
Photos: http://wahoo.no-ip.org/piwigo                   paka @ IRCnet oftc

**Comment by matt-hoskins** [comment]
> * matt-hoskins ***@***.***> [01-09-25 15:09]:
> Left-click-on-thumbnail-to-navigate isn't working for me either in the culling-layout-in-fixed-mode either - left clicking selects the thumbnail (the background to the thumbnail turns a lighter grey) but doesn't change what images are being displayed above (i.e. change in behaviour from 4.8).
> <left> or <right><arrow> will change image in the thumbnail display.
> […](#)
> -- (paka)Patrick Shanahan Plainfield, Indiana, USA @ptilopteri facebook/ptilopteri Photos: http://wahoo.no-ip.org/piwigo paka @ IRCnet oftc

Yeah, that still works - as does scrolling with the mouse wheel while hovering the pointer over the light table - and I can get by for now, but my typical use is that I have the fixed culling layout set to show two images side-by-side and if I want to jump about in a collection I use the thumbnail view below to quickly scroll through a collection to the next sequence I want to do culling on and then click on one of the images to jump to that sequence. I assume (hope) it's a regression losing the click-thumbnail-to-navigate in 5.0!
